_posts/zh/2019-06-14-exec-briefing.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bitcoin Optech 2019 高管简报
+permalink: /zh/2019-exec-briefing/
+name: 2019-06-14-exec-briefing-zh
+type: posts
+layout: post
+lang: zh
+slug: 2019-06-14-exec-briefing-zh
+
+excerpt: >
+  2019 年 5 月 14 日 Bitcoin Optech 高管简报的视频和幻灯片。
+
+auto_id: false
+---
+{% include references.md %}
+
+
+2019 年 5 月 14 日，Bitcoin Optech 在纽约的 Chaincode Labs 办公室举办了一次行政简报会议。 这些研讨会的目标是比特币公司高管和创始人，涵盖了我们认为 2019 年高管需要了解的最重要的技术主题：
+
+- [__手续费回落__](#手续费回落) - 为什么比特币手续费会激增，为什么高手续费可能会回落，以及企业如何在高手续费环境中蓬勃发展。
+- [__在闪电网络上运行__](#在闪电网络上运行) - 主网闪电网络容量在过去 12 个月内迅速增长。对当前闪电网络使用情况的调查以及比特币企业采用闪电网络的机会讨论。
+- [__下一个软分叉__](#下一个软分叉) - 即将到来的比特币软分叉提案的功能：Schnorr 签名、Taproot 和 SIGHASH_NOINPUT。这些变化对企业可能意味着什么。
+
+## 手续费回落
+
+第一场演讲由 Bitcoin Optech 贡献者 Mike Schmidt 进行，重点讨论了交易手续费以及降低成本和用户困惑的方式。
+
+[![手续费回落](/img/posts/2019-exec-briefing/fees.png)](https://www.youtube.com/watch?v=ihUQ4C42KUk)
+{:.center}
+
+[下载幻灯片][fees slides]
+{:.center}
+
+{% include specials/2019-exec-briefing/fees.md %}
```

Comment:
Do you want to translate these pieces of content or only translate the intro to this blog post?

Reply:
I checked that there are already translated files in path `specials/2019-exec-briefing/zh/`. I add `/zh/` in the path.